docs/user/metacat/source/authinterface.rst

```diff
@@ -1,112 +1,19 @@
 Metacat Authentication Mechanism
 ================================
-Metacat supports either an internal password file authentication or the use of LDAP 
-as an external authentication mechanism.  It does this by supplying two classes 
-(``AuthFile`` or ``AuthLDAP``) that implement authentication via a password file or 
-an external LDAP server. You may choose the authentication mechanism during initial configuration.
 
-If neither of these choices is suitable for your deployment, a custom authentication mechanism can be built.
-Metacat is written such that this Authentication provider is replaceable with 
-another class that implements the same interface (``AuthInterface``). As 
-an Administrator, you have the choice to provide an alternative implementation 
-of ``AuthInterface`` and then configuring **metacat-site.properties** to use that
-class for authentication instead of LDAP or the internal password file.
+Metacat only supports ORCID authentication as an authentication mechanism. File-based or LDAP
+authentication is no longer available. Registering for an ORCID is simple, please visit:
+  https://orcid.org/
 
-File-Based Authentication
-----------------------------------
-This is the default authentication mechanism in Metacat. The password file
-path can be specified during initial configuration. The Tomcat user should have 
-write/read permission to access the file. The password file follows this form:
+After signing up for an ORCID iD, you may use it as an admin identity when first configuring Metacat
+authentication settings. Note, your full ORCID iD includes `https://orcid.org/` not just the 16-digit
+ORCID iD:
+  ex. http://orcid.org/0000-0001-2345-6789
```

Comment:
You mention `https` on line 9, but show `http` on line 11.
I saw a note from @mbjones saying we should use https in the docs, but I can't find it now in order to discuss there, so adding a comment here... (and also flagging @taojing2002)

Both the instructions and the validation on the admin configuration page explicitly require orcids of the form http://, not https://, so I think it will be confusing if the documentation does not match the actual experience.

We have a couple of options:

1. change the admin config so it accepts either http or https, and then only uses http in the background (but do we save the https values so they still appear next time the user looks at them? and if so, we'll have to change the backend authorization cade to strip off the "s", which is fine, but this change, review and testing will delay the 3.0.0 release.)
2. Say something here in the docs about orcids treally being of the form "https" but our system requires you to enter them as http, and a note saying why that's not any less secure in this particular case
3. Some other option?

Reply:
My note on this is in the the README pull request #1778, essentially that ORCID specifies https for display, and we should use that everywhere except in actual config fields, where we use the http version because we started recording ORCID subjects in access rules before they switched to https. So we have a legacy problem for ORCID identity comparisons. We have a ticket somewhere about this, but I can't track it down at the moment.

Reply:
> https for display, and we should use that everywhere except in actual config fields, where we use the http version

That's the conundrum here -- this document is all about the actual config fields, so I think using https here will be confusing, unless we say what you said above.

Reply:
yeah, agreed. Our docs should use `https` for display in the READE, but in places like this where we are talking about config fields that hold ORCIDs for authentication, we should be clear that the http version is required.

Reply:
Understood, I have updated the URL `https://orcid.org/` to `http://orcid.org/ to keep these references consistent in the admin.

If users clicked on the string with `http`, the ORCID website has a redirect set up to `https://orcid.org/`.